1.3.1

Konicai opened the pull request:
Features:
- Add pitch/yaw values to Join-Teleporter
- Add option to enable/disable bstats
- Add `Respawn` value to access items, if they should be given when the player respawns
- Main config bump from version 5 to 6: Join-Teleporter will be automatically modified to the correct format. Bstats option must be added manually, or else it will be enabled by default.
- Selector config version *not* bumped: `Respawn` must be added manually to access item entries, or a default value of true will be used.


Misc:
- Broadcast, join message, scoreboard, and world settings are now disabled in the default config.
- Drop spigot target api from 1.16 to 1.14
- Bump Floodgate api version to 2.1.0
- Bump PlaceholderAPI version to 2.10.10

Bugfixes:
- Unnecessary warn messages that spammed
- Bad calculation when giving access items that resulted in the items sometimes not being given depending on the contents of the hotbar



resolves https://github.com/ProjectG-Plugins/GeyserHub/issues/22

Konicai committed:
Add pitch/yaw to join teleporter

Konicai committed:
fix resource filtering

Konicai committed:
update main config version in ConfigId

Konicai committed:
update config version in 4-5 main updater

Konicai committed:
logger level change

Konicai committed:
whoops

Konicai committed:
put yaw before pitch in config

Konicai committed:
bump spigot api to 1.17 in spigot.yml

Konicai committed:
Drop to spigot 1.14, bump floodgate and papi versions

Konicai committed:
improvements to command executor and java menu listener

Konicai committed:
Improvements to access items
- Major fixes to calculations when giving access items
- New access item config values "Respawn", if the item should be given on respawn

Jens-Co reviewed (approved):
All good for me! Nice work yet again

Konicai committed:
Merge remote-tracking branch 'origin/master' into 1.3.1

Konicai committed:
Various changes
- fix access item giving on join / respawn
- disable broadcast, join message, scoreboard, and world settings by default
- add option to disable bstats

Konicai committed:
update badges

Konicai committed:
fix access item giving on join
